### PR TITLE
Throw error when domain parameter is included in DENY & SAMEORIGIN

### PR DESCRIPTION
**Pull request by gzog**
Resolves https://github.com/helmetjs/frameguard/issues/17

### CODE REVIEW

File: test/index.test.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ describe('frameguard', () => {
       const str = new String('SAMEORIGIN'); // eslint-disable-line no-new-wrappers
       await request(app(frameguard({ action: str as any }))).get('/')
         .expect('X-Frame-Options', 'SAMEORIGIN');
-      expect(str.valueOf()).toBe( 'SAMEORIGIN');
+      expect(str.valueOf()).toBe('SAMEORIGIN');
```
Comment (EvanHahn) [inline test/index.test.ts]: Good catch—thanks! Surprised the linter didn't catch this...

File: index.ts [hunk 2]
```diff
@@ -57,6 +57,9 @@ function getHeaderValueFromOptions(options?: FrameguardOptions): string {
     const domain = parseDomainOption(options.domain);
     return `${action} ${domain}`;
   } else {
+    if ('domain' in options) {
```
Comment (EvanHahn) [inline index.ts]: Minor nitpick: could we change this to:

```js
if (action === 'ALLOW-FROM') {
  // ...
} else if ('domain' in options) {
  throw new Error(`domain parameter not supported for ${action}`);
} else {
  return action;
}
```

**Comment by gzog** [comment]
@EvanHahn Fixed that nitpick :)

**Comment by EvanHahn** [comment]
This has been merged into the `4.x` branch, which you can see [here](https://github.com/helmetjs/frameguard/pull/20).

I'm going to close this PR because it's been merged into another branch. Thanks!

**Pull request closed without merging**